packages/manager/apps/dedicated/client/app/index.html

```diff
@@ -72,9 +72,7 @@
                         data-sidebar-namespace="'account'"
                     ></div>
 
-                    <div
-                        class="col-md-9 col-md-offset-3 col-lg-10 col-lg-offset-2 px-0"
-                    >
+                    <div class="col-lg-10 col-lg-offset-2 px-0">
```

Comment:
You sure this works ? Did you try every screen resolution ? This issue is present a lot in the Managers

Reply:
Yes it looks good in all resolutions. col-md-9 is overwritten in main css with width for media queries